check_pods.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+NAMESPACE="$ns"
+if [[ "$NAMESPACE" == "" ]]; then
+    NAMESPACE=default
+fi
+if [[ "$APP_NAME" == "" ]]; then
+    APP_NAME=pythonservice
+fi
+if [[ "$RELEASE" == "" ]]; then
+    RELEASE="mesh-tutorial"
+fi
+
+export POD=$(kubectl get pods --namespace $NAMESPACE -l "app.kubernetes.io/name=$APP_NAME,app.kubernetes.io/instance=$RELEASE" -o jsonpath="{.items[0].metadata.name}")
+export READY=$(kubectl get pod $POD -o jsonpath='{.status.containerStatuses[0].ready}')
+
+echo "Checking pod $POD of"
```

Comment:
This message does not read very well...
Suggested change:
```
echo "Checking ready status of pod $POD:
echo "  RELEASE:      $RELEASE"
echo "  APP_NAME:     $APP_NAME"
echo "  NAMESPACE:    $NAMESPACE"
echo "  READY STATUS: $READY"
```

and further down inside the while loop, change it like this so that it matches the output above:
```
    echo "  READY STATUS: $READY"
```